task5/src/main/java/com/clouway/adapter/http/BankOperationHandler.java

```diff
@@ -54,13 +57,15 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 
     if (operation.equalsIgnoreCase("Deposit")) {
       fundsRepository.deposit(amountAsDouble, email);
+      fundsRepository.updateHistory(System.currentTimeMillis(), email, operation, amountAsDouble);
```

Comment:
There is no need to pass the date as param. The advice that I gave you was to get the time as miliseconds into the `updateHistory()`. There can be situation that you can pass it like now but your case is different.